tests/providers/amazon/aws/operators/test_emr_system.py

```diff
@@ -0,0 +1,33 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from tests.test_utils.amazon_system_helpers import AWS_DAG_FOLDER, AmazonSystemTest
+
+
+class EmrSystemTest(AmazonSystemTest):
+    """
+    System tests for AWS EMR operators
+    """
+    @classmethod
+    def setup_class(cls):
+        cls.create_emr_default_roles()
```

Comment:
Shouldn't we remove these default roles at the end? So that you can call this tests over and over.
I also used [pytest fixtures](https://docs.pytest.org/en/latest/fixture.html).

Reply:
I don't think it's necessary to remove these default roles for running the tests. The command `aws emr create-default-roles` will only take effect if the associated roles (Service Role) are not set. Otherwise, it will do nothing.

Reply:
Also, to fully clean up the default roles created, it involves six API calls. I guess that in the future, there will be a dedicated managed AWS account for running system tests. So we can simply reuse the default roles.

Reply:
For pytest fixtures, it seems that all shared fixtures should go into the `contest.py` ( I cannot find this to be configurable at first glance. Please let me know if I am wrong). So it does not quite fit with how we store our fixtures at the moment (a mix of xunit-style setup and pytest fixtures - maybe we can unify it in the future?). And I think the current code in this PR provide the least boilerplate code

Reply:
Thanks for the explanation. Sounds good to me.